src/stem-tracker.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+var Stem = require("./stem");
+
+/**
+ * Stem Tracking Utility.
+ *
+ * Handles the stems for a given editor.
+ */
+module.exports = function StemTracker(editor, onClick) {
+  var me = this;
+  var previousStems = [];
+  var stems = [];
+  var topLevelTags = ['p', 'h1', 'h2', 'h3', 'div', 'section'];
+
+  // Listen for changes in content and update.
+  editor.addListener({
+    onContent: function () {
+      updateStems( editor.currentElem() );
+      return false;
+    },
+    onKey: function(e) {
```

Comment:
delete this? it shouldn't be necessary. if it is then that means onContent isn't being called enough.